.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+language: ruby
+rvm:
+  - 2.3.1
+services:
+  - docker
```

Comment:
If we aren't already, we should (eventually) run the tests in the Docker container.